src/shiftedRootNormLhalf.jl

```diff
@@ -0,0 +1,61 @@
+export ShiftedRootNormLhalf
+
+mutable struct ShiftedRootNormLhalf{
+  R <: Real,
+  V0 <: AbstractVector{R},
+  V1 <: AbstractVector{R},
+  V2 <: AbstractVector{R},
+} <: ShiftedProximableFunction
+  h::RootNormLhalf{R}
+  xk::V0
+  sj::V1
+  sol::V2
+  shifted_twice::Bool
+
+  function ShiftedRootNormLhalf(
+    h::RootNormLhalf{R},
+    xk::AbstractVector{R},
+    sj::AbstractVector{R},
+    shifted_twice::Bool,
+  ) where {R <: Real}
+    sol = similar(xk)
+    new{R, typeof(xk), typeof(sj), typeof(sol)}(h, xk, sj, sol, shifted_twice)
+  end
+end
+
+shifted(h::RootNormLhalf{R}, xk::AbstractVector{R}) where {R <: Real} =
+  ShiftedRootNormLhalf(h, xk, zero(xk), false)
+shifted(
+  ψ::ShiftedRootNormLhalf{R, V0, V1, V2},
+  sj::AbstractVector{R},
+) where {R <: Real, V0 <: AbstractVector{R}, V1 <: AbstractVector{R}, V2 <: AbstractVector{R}} =
+  ShiftedRootNormLhalf(ψ.h, ψ.xk, sj, true)
+
+fun_name(ψ::ShiftedRootNormLhalf) = "shifted Lhalf norm"
+fun_expr(ψ::ShiftedRootNormLhalf) = "t ↦ ‖xk + sk + t‖ₚᵖ, p = 1/2"
+fun_params(ψ::ShiftedRootNormLhalf) = "xk = $(ψ.xk)\n" * " "^14 * "sj = $(ψ.sj)"
+
+function prox!(
+  y::AbstractVector{R},
+  ψ::ShiftedRootNormLhalf{R, V0, V1, V2},
+  q::AbstractVector{R},
+  σ::R,
+) where {R <: Real, V0 <: AbstractVector{R}, V1 <: AbstractVector{R}, V2 <: AbstractVector{R}}
+  γλ = 2 * σ * ψ.λ
+  ϕ(z) = acos(γλ / 8 * (abs(z) /3 )^(-3/2))
```

Comment:
```suggestion
  ϕ(z) = acos(2γλ / 8 * (abs(z) /3 )^(-3/2))
```

Reply:
Any reason why this one isn't `γλ / 4`?

Reply:
It could be. If γλ = 2 * σ * ψ.λ then it isn't, but if it doesn't contain the 2*() then it can be 4.